packages/block-library/src/cover/edit.js

```diff
@@ -279,6 +282,11 @@ class CoverEdit extends Component {
 					{}
 			),
 			backgroundColor: overlayColor.color,
+			...(
+				customGradient && ! url ?
+					{ background: customGradient } :
+					{}
```

Comment:
Minor: I do prefer setting keys conditionally afterwards, but maybe that's just me. It makes the spreading unnecessary.